src/Og.php

```diff
@@ -604,23 +606,6 @@ public static function groupManager() {
   }
 
   /**
-   * Get a role by the group's bundle and role name.
-   *
-   * @param string $entity_type_id
-   *   The group entity type ID.
-   * @param string $bundle
-   *   The group bundle name.
-   * @param string $role_name
-   *   The role name.
-   *
-   * @return \Drupal\og\OgRoleInterface|null
-   *   The OG role object, or NULL if a matching role was not found.
-   */
-  public static function getRole($entity_type_id, $bundle, $role_name) {
```

Comment:
OK will restore it! I will make a followup to move it to `OgRole`, because it will need some discussion about the correct naming of the function. Burying this in here will cause it to be missed by everyone who is involved in OG atm.